lvsfunc/util.py

```diff
@@ -102,6 +103,45 @@ def get_prop(frame: vs.VideoFrame, key: str, t: Type[T]) -> T:
     return prop
 
 
+def select_frames(clips: Union[vs.VideoNode, List[vs.VideoNode]],
+                  indicies: Union[List[int], List[Tuple[int, int]]]) -> vs.VideoNode:
```

Comment:
i'm not really sure what the point of unioning indices with `List[int]` is, it does make the function a bit more ergonomic but in reality indices should almost always be generated not directly user-entered

Reply:
I disagree with this point - The function is not intended to be called by the end user, however the flexibility of the function is part of the point of it's design. Why be less flexible if I don't have to? In any case, rfs ect. are intended to be the external functions, but if the user wishes to do their own thing then they can. If you have a better idea of what types these arguments could be feel free to suggest, though I think that writing even dataclasses for this would be over the top for something so simple.